### DISCUSSION TITLE
2023-11-10

### DISCUSSION THREAD

**Discussion by miniposi** [10-rolling/rollingPolling#57] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] post page 구현
- [x] 윤혁님 sendMessage 버튼 수정

### 📝 회의내용

* 해온 거 발표
* 수영님 피드백
* 멘토링 때 여쭤볼 내용 -> 각자 노션에 적기
* 오늘 일정 간략하게 말하기 ><

### 💪🏻 오늘 할일

* 각자 하고 싶은 거 하기
* 윤혁님 수정 사항 merge
* 수영님 - 무한 스크롤 구현
* 윤혁님 - Readme.md 간지나게 만들기
* 슬희님 - list page 만들기
* 궁수 - 수영님 에러 돕기! & 저번 구현 피드백 후 머지

### 🤔 특이사항

* 너무 열심히 하시네요들

**Comment by hanseulhee** [comment]
각자 하고 싶은 거 하기 .....? 그럼 전 놀겠습니다 !!

**Reply by miniposi** [reply]
곤란한데요

**Reply by oauch** [reply]
대단하시네요

**Reply by hanseulhee** [reply]
다들 감사합니다 ㅎㅎ
